src/webui/www/private/scripts/client.js

```diff
@@ -702,6 +702,18 @@ window.addEventListener("DOMContentLoaded", (event) => {
                 case TRACKERS_WARNING:
                     span.lastElementChild.src = "images/tracker-warning.svg";
                     break;
+                default: {
+                    if (LocalPreferences.get("show_tracker_favicon", "true") === "false")
+                        break;
+                    const link = host.split(".").slice(1).join(".");
+                    const img = trackerFilterItem.getElementsByTagName("img")[0];
```

Comment:
Just a random idea. Is it possible to use an `iframe` instead of `img`? and sandbox it properly.

Reply:
This should work:
```js
<iframe sandbox srcdoc="<img src=&quot;https://www.kali.org/images/favicon.ico&quot;>"></iframe>
```